sdk/storage/storage-blob/karma.conf.js

```diff
@@ -64,7 +64,8 @@ module.exports = function(config) {
       "MD_ACCOUNT_NAME",
       "MD_ACCOUNT_SAS",
       "ENCRYPTION_SCOPE_1",
-      "ENCRYPTION_SCOPE_2"
+      "ENCRYPTION_SCOPE_2",
+      "BLOB_VERSIONING_ENABLED",
```

Comment:
I guess this is temporary and will be removed once the feature is available?

Reply:
Yes. We will configure the ARM template to turn on this feature.